lib/README.md

```diff
@@ -23,13 +23,13 @@ Reduce up to 40% your Cypress suite execution time parallelizing the test run on
 ## Install
```

Comment:
Just copied this from `./README.md` to match. Do we need both, or can we delete 1?